src/README.md

```diff
@@ -82,19 +82,19 @@ already installed, the workflow goes like this:
   * When you see "Rocket has launched from http://...." you should be
     able to connect to the running app
   * Assuming you run the app on `localhost` and use the `7878` port, you
-    can browse to `http://localhost:7878/auth/sign_up` and register a
```

Comment:
Reminder to restore /auth here (and 2 more cases below).

Reply:
oops ! :D

Reply:
Okay... I JUST verified that you can simply browse to localhost:7878 and it will automatically append the auth.